lib/PdfScraper.py

```diff
@@ -0,0 +1,33 @@
+from io import StringIO
+
+from pdfminer.converter import TextConverter
+from pdfminer.layout import LAParams
+from pdfminer.pdfinterp import PDFResourceManager, PDFPageInterpreter
+from pdfminer.pdfpage import PDFPage
+
+
+class PdfScraper:
```

Comment:
I'm still figuring out what idiomatic python looks like in terms of classes vs functions. This will likely become just a function again since there's nothing in the constructor specific to a given instance. I have an idea for what to do next with this class that will make more sense, soon.